### DISCUSSION TITLE
Weblate doesn't update the repo

### DISCUSSION THREAD

**Discussion by alexeysvyridov** [WeblateOrg/weblate#6494] [Q&A] [answered]
When I click pull or update the repo doesn't update.
I don't get any errors.

What happened, and how I can know what actually happened and how to fixed ?
![image](https://user-images.githubusercontent.com/33163048/131451679-de161b50-7fbe-476d-be02-80b6f90451bb.png)
![image](https://user-images.githubusercontent.com/33163048/131451725-bd169d0b-5703-4ca0-970c-0938112019a0.png)

**Comment by nijel** [comment, marked as answer]
Don't you get a merge failure? It should be visible at least in the repository changes panel.
